javascript/javascript3/week1/weather/weather.js

```diff
@@ -0,0 +1,61 @@
+const button = document.getElementById("button1");
+const input = document.getElementById("userinput");
+
+let body = document.querySelector("body");
+let ul = document.querySelector("ul");
+
+//const apiKey = "c65dee63b1543ccd79345b5fdaa81ee6";
+
+const url = `https://api.openweathermap.org/data/2.5/weather?q=copenhagen&appid=c65dee63b1543ccd79345b5fdaa81ee6&units=metric`;
```

Comment:
what is the benefit of taking user input if you are hardcoding city here = copenhagen, Make it dynamic.